infra/infra.go

```diff
@@ -38,13 +38,17 @@ func main() {
 		Short('a').
 		StringVar(&g.Auth)
 	k8sGKE.Flag("file", "yaml file or folder  that describes the parameters for the object that will be deployed.").
-		Required().
 		Short('f').
 		ExistingFilesOrDirsVar(&g.DeploymentFiles)
 	k8sGKE.Flag("vars", "When provided it will substitute the token holders in the yaml file. Follows the standard golang template formating - {{ .hashStable }}.").
 		Short('v').
 		StringMapVar(&g.DeploymentVars)
 
+	k8sGKEShow := k8sGKE.Command("show", "show GKE related infomation").
```

Comment:
We should probably just have one command, also a short flag would be nice. :sparkles: